custom/icds_reports/ucr/expressions.py

```diff
@@ -781,3 +781,36 @@ def get_app_version(spec, context):
 
 def datetime_now(spec, context):
     return DateTimeNow.wrap(spec)
+
+
+def get_related_docs_ids(case_id):
+    """gets a related case id from an ICDS ccs_record or child_health case. Expects the hierarchy:
+                                      /---->  ccs_record
+         household -----> person  --<
+                                      \---->  child_health
+
+    This could likley be optimized pretty easily to take in multiple case_ids but we
+    don't currently have an interface to do so
+    """
+    db = CaseAccessors('icds-cas')
+    current_case = db.get_case(case_id)
+    if not current_case or not current_case.indices:
+        return []
+    person_case = db.get_case(current_case.indices[0].referenced_id)
```

Comment:
seems like you are making assumptions about the case here that should be checked? e.g. that its type is one of the expected ones.